apps/base-docs/docs/building-with-base-and-foundry/testing.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Testing with Foundry
+description: Documentation for Foundry, a toolchain for smart contract development. Learn how to test your smart contracts using Foundry.
+keywords: [Foundry, Forge, Foundry Book, smart contract development, toolchain, testing, test]
+hide_table_of_contents: false
+---
+
+In this article, you'll learn how to test your smart contracts using [Foundry], the toolchain for smart contract development.
+
+---
+
+## Objectives
+
+By the end of this guide, you should be able to:
+
+- Understand the importance of testing in smart contract development.
+- Write and execute tests for smart contracts using the Forge Standard Library with Foundry.
+- Use some of the `cheatcodes` that Foundry provides.
+
+## Overview
+
+Testing is a crucial aspect of smart contract development, ensuring the reliability and security of your code. Foundry provides a robust testing framework that allows developers to create comprehensive test suites for their projects using Solidity.
+
+## My First Test with Foundry
+
+Considering the default test that the `forge init hello_foundry_in_base` command provides in the seed Foundry project.
+
+```javascript
```

Comment:
I believe we support solidity code blocks if you change `javascript` to `solidity`.

(Apply to all solidity code blocks)

https://github.com/base-org/web/blob/master/apps/base-docs/docusaurus.config.js#L157C40-L157C40